libyul/Object.h

```diff
@@ -68,7 +68,15 @@ struct Object: ObjectNode
 
 	/// @returns the set of names of data objects accessible from within the code of
 	/// this object.
-	std::set<YulString> dataNames() const;
+	std::set<YulString> qualifiedDataNames() const;
+
+	/// @returns tuple consisting of
+	/// 	bool true if _qualifiedName is referring to data
+	/// 	vector of subIDs if possible to reach subobject with @_qualifiedName, empty vector otherwise
```

Comment:
What does if possible" mean? If the name cannot be found, does it return an empty vector or does it throw an assertion failure?